tests/storage/export.go

```diff
@@ -591,6 +591,49 @@ var _ = Describe(SIG("Export", func() {
 		Entry("with RAW kubevirt content type block PROXY", decorators.RequiresBlockStorage, populateKubeVirtContent, verifyKubeVirtRawContent, libstorage.GetRWOBlockStorageClass, createCaConfigMapProxy, urlGeneratorProxy, exportv1.KubeVirtRaw, kubevirtcontentUrlTemplate, k8sv1.PersistentVolumeBlock),
 	)
 
+	It("should make sure PVC export is Ready when source pod is Completed", func() {
+		sc, exists := libstorage.GetRWOFileSystemStorageClass()
+		if !exists {
+			Fail("Fail test when right storage is not present")
+		}
+		volumeMode := k8sv1.PersistentVolumeFilesystem
+		By("Creating source DV with a pod that reatin after completion")
+		dv := libdv.NewDataVolume(
+			libdv.WithAnnotation("cdi.kubevirt.io/storage.pod.retainAfterCompletion", "true"),
```

Comment:
Don't really need this annotation. It is used to keep the importer pod around after migration so we can inspect it during debugging or if we are looking for a particular log message. In this case we are doing neither.